index.yaml

```diff
@@ -73,6 +73,13 @@ distributions:
     source_builds:
       default: melodic/source-build.yaml
       ds: melodic/source-stretch-build.yaml
+  noetic:
+    ci_builds:
+      bionic-python2: noetic/ci-bionic-python2.yaml
+      bionic-python3: noetic/ci-bionic-python3.yaml
+    notification_emails:
+    - ros-buildfarm-noetic@googlegroups.com
+    - sloretz+buildfarm@openrobotics.org
```

Comment:
@sloretz I just guess the email address you might want to use for notifications based on what we used in past distros. Please feel free to adjust it to your preference.

Reply:
Emai LGTM